evm/test/AggregatorProxy_test.js

```diff
@@ -0,0 +1,119 @@
+import * as h from './support/helpers'
+import { assertBigNum } from './support/matchers'
+const personas = h.personas
+
+contract('AggregatorProxy', () => {
+  const SOURCE_PATH = 'AggregatorProxy.sol'
+  const jobId1 =
+    '0x4c7b7ffb66b344fbaa64995af81e355a00000000000000000000000000000001'
+  const deposit = h.toWei('100')
+  const basePayment = h.toWei('1')
+  const response = 54321
+  const response2 = 67890
+  let link, aggregator, aggregator2, oc1, proxy
+  let jobIds = [jobId1]
+
+  beforeEach(async () => {
+    link = await h.linkContract()
+    oc1 = await h.deploy('Oracle.sol', link.address)
+    aggregator = await h.deploy(
+      'ConversionRate.sol',
+      link.address,
+      basePayment,
+      1,
+      [oc1.address],
+      [jobId1]
+    )
+    await link.transfer(aggregator.address, deposit)
+
+    proxy = await h.deploy(SOURCE_PATH, aggregator.address)
+  })
+
+  it('has a limited public interface', () => {
+    h.checkPublicABI(artifacts.require(SOURCE_PATH), [
+      'aggregator',
+      'currentAnswer',
+      'setAggregator',
+      // Ownable
```

Comment:
This should be removed or uncommented. It's present in `ConversionRate_test.js`, too.

Reply:
It's not actually a method, just delimiting the methods that are inherited from Ownable.sol vs defined in the contract.

Reply:
Oh, right. Maybe change the comment to `// Ownable methods:`, then.